### PR TITLE
Bookmarks ToolBar refresh fix

### PR DESCRIPTION
**Pull request by TARPTAEYA**

### CODE REVIEW

File: src/lib/bookmarks/bookmarkstoolbar.h [hunk 1]
```diff
@@ -57,6 +57,8 @@ private slots:
 
     void dropEvent(QDropEvent* e);
     void dragEnterEvent(QDragEnterEvent* e);
+    
+    int fixedMinHeight = -1;
```
Comment (nowrep) [inline src/lib/bookmarks/bookmarkstoolbar.h]: Style: `m_` prefix, also it should be added at the end.

File: src/lib/bookmarks/bookmarkstoolbar.cpp [hunk 2]
```diff
@@ -213,8 +213,13 @@ void BookmarksToolbar::addItem(BookmarkItem* item)
     button->setShowOnlyIcon(m_bookmarks->showOnlyIconsInToolbar());
     button->setShowOnlyText(m_bookmarks->showOnlyTextInToolbar());
     m_layout->addWidget(button);
-
-    setMinimumHeight(minimumSizeHint().height());
+    
+    if(fixedMinHeight != -1){
```
Comment (nowrep) [inline src/lib/bookmarks/bookmarkstoolbar.cpp]: Style `if (fixedMinHeight != -1) {`

**Comment by TARPTAEYA** [comment]
yeah, only one condition would be sufficient, fixed :)

File: src/lib/bookmarks/bookmarkstoolbar.cpp [hunk 3]
```diff
@@ -213,8 +213,11 @@ void BookmarksToolbar::addItem(BookmarkItem* item)
     button->setShowOnlyIcon(m_bookmarks->showOnlyIconsInToolbar());
     button->setShowOnlyText(m_bookmarks->showOnlyTextInToolbar());
     m_layout->addWidget(button);
-
-    setMinimumHeight(minimumSizeHint().height());
+    
+    if(!m_fixedMinHeight){
```
Comment (nowrep) [inline src/lib/bookmarks/bookmarkstoolbar.cpp]: Spaces: `if (!m_fixedMinHeight) {`

Comment (TARPTAEYA) [reply src/lib/bookmarks/bookmarkstoolbar.cpp]: Yeah fixed, seems like I must go through code formatting first :grin:

**Pull request merged**